README.md

```diff
@@ -46,6 +46,15 @@ Pass the path to configuration files on the command line with `-c` or `--config-
 If multiple configuration files are provided, they will be merged together.
 If a configuration key occurs in more than one configuration file, the process will exit with an error.
 
+By default, a node will not expose any RPC methods.
+To enable APIs, you must set `enabled_apis` under the [[nodes]] object in the configuration file.
+Each item in `enabled_apis` must either be:
+
+* A string such as `"eth"`, which enables all API methods under the `eth_` namespace.
+* An object of the form `{ namespace = "eth", apis = ["blockNumber"] }`, which enables specific API methods.
```

Comment:
Is this the right level of granularity for enabling APIs? This lets you enable an entire namespace or specific API methods. One option that is deliberately missing is "Enable all APIs in this namespace, except for these". Is this okay?

Reply:
It would be useful to be able to disable only some methods. Happy to merge as is though.